drivers/iio/adc/Kconfig

```
@@ -461,6 +461,17 @@ config ADAQ8092
 	  To compile this driver as a module, choose M here: the
 	  module will be called adaq8092.
 
+
+config ADA4355
+	tristate "Analog Devices ADA4355 uModule Data Acquisition Module"
+	depends on SPI
```

```suggestion
	depends on SPI
	depends on CF_AXI_ADC
```

The driver uses axiadc_write and axiadc_read, exported from cf_axi_adc_core.c, enabled with CF_AXI_ADC.
So the driver depends on this symbol as well.
Without this change, it cannot be compiled as a module without blindly enabling symbols.
This is an issue in other drivers.